webclient/webclient/src/main/java/io/helidon/webclient/WebClientRequestBuilderImpl.java

```diff
@@ -95,6 +95,11 @@
     static final AttributeKey<ConnectionIdent> CONNECTION_IDENT = AttributeKey.valueOf("connectionIdent");
     static final AttributeKey<Long> REQUEST_ID = AttributeKey.valueOf("requestID");
 
+    /**
+     * Whether the channel will be closed and keep-alive caching should not be applied.
+     */
+    static final AttributeKey<Boolean> WILL_CLOSE = AttributeKey.valueOf("willClose");
```

Comment:
Shouldn't we use some kind of Helidon specific prefix for attributes? Are there any reserved names by Netty?

Reply:
I don't think there are any to be fair. Or at least I haven't heard of any. I can check if you want.